packages/server/src/services/agent.service.ts

```
@@ -96,6 +109,93 @@ export class AgentService implements IAgentService {
     }
   }
 
+  async *handleUserRequestWebsocket(
+    agent: AgentSystem | any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)